back/queries/adventure.js

```diff
@@ -38,6 +38,25 @@ async function UserReviewsQuery(username) {
   ).map((review) => review.toJSON());
 }
 
+/* Find all reviews of an experience by ID
+   It groups those made by the same user, since each review is on a single
+   preference area
+Used by functions/filterAndRank */
+async function AllExperienceReviewsQuery(experienceID) {
+  let totalReviews = 0;
+  const ratingUsers = [];
+  const reviews = await new Parse.Query('Review')
+      .equalTo('experienceId', experienceID)
+      .find();
+  reviews.map((review) => {
+        if (!ratingUsers.includes(review.toJSON().adventurerUsername)) {
```

Comment:
The formatting looks a bit off here. BTW you can also set up prettier to run at other points in time, such as [when you run git add](https://prettier.io/docs/en/precommit.html). These are called precommit hooks.

Reply:
That is great! Working on it!